ocs_ci/templates/ocs-deployment/toolbox.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: rook-ceph-tools
-        image: {{ rook_image | default('rook/ceph:v0.9.0-519.g111610e') }}
+        image: {{ rook_image | default('rook/ceph:v1.1.1') }}
```

Comment:
Once we will clean up all the properties needed for old deployment we would like to remove also the rook_image from ENV_DATA section.  We can follow up in another PR but we should take the rook version dynamically from CSV. So this case we will make sure we are using the same version from the build.

Reply:
Just for sure, currently this image will be used:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L84
which is 'rook/ceph:master' .
Would we like to switch now to v1.1.1 or master image?   The dynamic set of this parameter will help us to be consistent with what do we have deployed.

Reply:
Its now using the version that rook-operator has.

Reply:
Not sure you got me @vasukulkarni , but I wanted to say that if you changing the default value here in template, as we passing env data from the configuration when rendering the template this default value defined in template is overwritten by this value https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L84 mentioned in previous config.  So the master will be used and not v1.1.1 .  And I think you wanted to use master.   And we should take this value dynamically as we want to get rid of this value in our config as I pointed. But this can be done in different PR.

Reply:
I see what you are saying,  I will discuss with you in chat , we have 2 toolbox yaml, one that uses rendering and one that doesn't. I used the other one.  Probably we should get rid of the ceph version in the config now, what I noticed with upstream operator is operator decides the version, I think with latest master its still v1.1.1